README.md

```diff
@@ -127,127 +127,175 @@ future state of the libraries and to discover projects for contribution.
-would defer to the shepherd to do the first few rounds of reviews, after which
-the rest of the team should be engaged.
-
-## RFC Life Cycle
+To kick off an RFC process, simply create a [new tracking issue] and follow the
+instructions in the template. The issue includes a checklist of the various
+stages an RFC goes through.
+
+[new tracking issue]: https://github.com/aws/aws-cdk-rfcs/issues/new?assignees=&labels=management%2Ftracking%2C+status%2Fproposed&template=tracking-issue.md&title=proposal+title
+
+This section describes each stage in detail, so you can refer to it for
+guidance.
+
+### 1. Create a tracking issue
+
+Each RFC has a GitHub issue that tracks it from start to finish. The issue is
+the hub for conversations, community signal (+1s) and the issue number is used
+as the unique identifier of this RFC.
+
+> Before creating a tracking issue, please search for similar or related ideas in
+the RFC table above or in the issue list of this repo. If there is a relevant
+RFC, collaborate on that existing RFC, based on its current stage.
+
+Our [tracking issue template] includes a checklist of all the steps an RFC goes
+through and it's the driver's responsibility to update the checklist and assign
+the correct label to on the RFC throughout the process.
+
+[tracking issue template]: https://github.com/aws/aws-cdk-rfcs/blob/master/.github/ISSUE_TEMPLATE/tracking-issue.md
+
+When the issue is created, it is required to fill in the following information:
+
+1. **Title**: the name of the feature or change - think changelog entry.
+2. **Description**: a _short_ description of feature, as if it was already implemented.
+3. **Proposed by**: fill in the GitHub alias of the person who proposed the idea
+   under "Proposed by".
+
+### 2. Identify roles and responsibilities
+
+Each RFC has a __driver__, __approver(s)__ and __stakeholders__.
+
+The __driver__ is the person responsible to drive this RFC to completion. This
+does __not__ mean that the driver must be the person actually doing all the
+work, but it is the person responsible to get the RFC through the finish line.
+
+__Approvers__ are the people that have the authority to approve the RFC and any
+subsequent changes to it. Approvers have the final word if we cannot reach
+consensus.
+
+__Stakeholders__ are people (internal/external) who may be impacted by this RFC
+or work on similar domains or projects. The goal is to make sure that
+stakeholders have an opportunity to influence the direction of the RFC in
+various stages of the process.
+
+The driver is responsible to update the **Roles** table in the tracking issue.
+
+### 3. Organize a kick-off meeting
+
+Before diving into writing the RFC, is it highly recommended to organize a
+kick-off meeting that includes the driver, approver(s) and stakeholders. The
+goal of the meeting is to discuss the feature, its scope and general direction
+for implementation.
+
+Our experience shows that such a meeting can save a lot of time and energy.
```

Comment:
My suggestion would be to do something different. It would be to identify 'some' high level approaches, put that down in the RFC or tracking issue, and then have that kickoff meeting. 

This way, there is some starting point and people don't arrive at a meeting with a blank slate.

Reply:
I'll add a recommendation to write something up in the tracking issue as a prep for that meeting